### PR TITLE
rename and update keymap for handwired/evk in the QMK configurator

### PR DESCRIPTION
**Pull request by YangPiCui**
title says all :)

### CODE REVIEW

**Comment by YangPiCui** [comment]
https://github.com/qmk/qmk_firmware/pull/13094

File: public/keymaps/h/handwired_evk_v0_1_x_default.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "keyboard": "handwired/evk/v0_1_x",
+  "keymap": "default",
+  "commit": "804d5c1c5d59d9a12c1d793289ccbd59cb650ec2",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_GRV",        "KC_ESC",   "KC_PSCR",   "TG(1)",    "KC_CAPS",        "KC_SLSH",  "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_evk_v0_1_x_default.json:8]:
```suggestion
      "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_GRV",        "KC_ESC",   "KC_PSCR",   "TG(1)",    "KC_CAPS",        "KC_BSLS",  "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",
```

File: public/keymaps/h/handwired_evk_v0_1_x_default.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      "KC_Q",     "KC_W",     "KC_E",     "KC_R",     "KC_T",     "KC_LBRC",       "KC_F1",    "KC_F2",     "KC_F3",    "KC_F4",          "KC_RBRC",  "KC_Y",     "KC_U",     "KC_I",     "KC_O",     "KC_P",
+      "KC_A",     "KC_S",     "KC_D",     "KC_F",     "KC_G",     "KC_TAB",       "KC_F5",    "KC_F6",     "KC_F7",    "KC_F8",           "KC_QUOT",  "KC_H",     "KC_J",     "KC_K",     "KC_L",     "KC_SCLN",
+      "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",                      "KC_F9",    "KC_F10",    "KC_F11",   "KC_F12",                     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_BSLS",
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_evk_v0_1_x_default.json:11]:
```suggestion
      "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",                      "KC_F9",    "KC_F10",    "KC_F11",   "KC_F12",                     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_SLSH", 
```

File: public/keymaps/h/handwired_evk_v0_1_x_default.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      "KC_A",     "KC_S",     "KC_D",     "KC_F",     "KC_G",     "KC_TAB",       "KC_F5",    "KC_F6",     "KC_F7",    "KC_F8",           "KC_QUOT",  "KC_H",     "KC_J",     "KC_K",     "KC_L",     "KC_SCLN",
+      "KC_Z",     "KC_X",     "KC_C",     "KC_V",     "KC_B",                      "KC_F9",    "KC_F10",    "KC_F11",   "KC_F12",                     "KC_N",     "KC_M",     "KC_COMM",  "KC_DOT",   "KC_BSLS", 
+      "KC_LALT",  "KC_LSFT",  "KC_DEL",   "KC_EQL",    "KC_LGUI",  "KC_LCTL",      "KC_HOME",  "KC_UP",     "KC_END",   "KC_PGUP",       "KC_RCTL",  "KC_RGUI",   "KC_MINS",  "KC_BSPC",  "KC_RSFT",  "KC_RALT",
+	              "KC_LEFT",  "KC_RGHT",               "KC_SPC",   "KC_ENT",       "KC_LEFT",  "KC_DOWN",   "KC_RGHT",  "KC_PGDN",       "KC_ENT",   "KC_SPC",                "KC_UP",    "KC_DOWN" 
+    ],
+    [
+      "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_GRV",        "KC_ESC",   "KC_PSCR",   "TG(1)",    "KC_CAPS",        "KC_SLSH",  "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_evk_v0_1_x_default.json:16]:
```suggestion
      "KC_1",     "KC_2",     "KC_3",     "KC_4",     "KC_5",     "KC_GRV",        "KC_ESC",   "KC_PSCR",   "TG(1)",    "KC_CAPS",        "KC_BSLS",  "KC_6",     "KC_7",     "KC_8",     "KC_9",     "KC_0",
```

File: public/keymaps/h/handwired_evk_v0_1_x_default.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      "KC_Q",     "KC_H",     "KC_R",     "KC_F",     "KC_X",     "KC_LBRC",       "KC_F1",    "KC_F2",     "KC_F3",    "KC_F4",          "KC_RBRC",  "KC_QUOT",  "KC_Y",     "KC_U",     "KC_O",     "KC_J",
+      "KC_L",     "KC_S",     "KC_N",     "KC_T",     "KC_D",     "KC_TAB",       "KC_F5",    "KC_F6",     "KC_F7",    "KC_F8",           "KC_SCLN",  "KC_P",     "KC_I",     "KC_A",     "KC_E",     "KC_K",
+      "KC_Z",     "KC_M",     "KC_C",     "KC_V",     "KC_B",                      "KC_F9",    "KC_F10",    "KC_F11",   "KC_F12",                     "KC_W",     "KC_G",     "KC_COMM",  "KC_DOT",   "KC_BSLS",
```
Comment (noroadsleft) [inline public/keymaps/h/handwired_evk_v0_1_x_default.json:19]:
```suggestion
      "KC_Z",     "KC_M",     "KC_C",     "KC_V",     "KC_B",                      "KC_F9",    "KC_F10",    "KC_F11",   "KC_F12",                     "KC_W",     "KC_G",     "KC_COMM",  "KC_DOT",   "KC_SLSH", 
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by YangPiCui** [comment]
Thanks for the commit suggestion! It was a good catch.

**Review by noroadsleft — COMMENTED** [review]
I'll have one more suggestion once the `qmk_firmware` PR is merged.

**Comment by yanfali** [comment]
@noroadsleft this still relevant?

**Comment by noroadsleft** [comment]
@yanfali The requisite PR (https://github.com/qmk/qmk_firmware/pull/13094) was closed as stale due to modified submodules. If the PR is reworked to fix that this can be reopened.

**Pull request closed without merging**